src/schema/v2/me/conversation/conversationEventConnection/conversationOrderEvents.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLID,
+  GraphQLString,
+  GraphQLEnumType,
+  GraphQLBoolean,
+  GraphQLUnionType,
+} from "graphql"
+import date from "schema/v2/fields/date"
+import { MessageType } from "schema/v2/me/conversation/message"
+import { NodeInterface } from "schema/v2/object_identification"
+import { ResolverContext } from "types/graphql"
+import gql from "lib/gql"
+
+/**
+ * fetch all order events for a conversation from exchange
+ * @param conversationId
+ * @param ctx
+ * @param { sellerId } - to authenticate as seller instead of buyer
+ * @returns
+ */
+export const fetchConversationOrderEvents = async (
+  conversationId: string,
+  // from ctx
+  { exchangeLoader, userID }: { exchangeLoader?: any; userID?: string },
+  // from args - presence of sellerId overrides buyer mode
+  sellerId?: string
+) => {
+  if (!exchangeLoader) return null
+  // this id changes for the buyer or seller
+  const viewerKey = sellerId
+    ? `sellerId: "${sellerId}"`
+    : `buyerId: "${userID}"`
+
+  const exchangeData = await exchangeLoader({
```

Comment:
I wonder if it might be helpful to add `GraphQL` to graphql loaders to distinguish them from other more common rest loaders -- so like `exchangeGraphQLLoader` 🤔